### ISSUE TITLE
JupyterLab 1.04 not supported?

### ISSUE THREAD

**Issue by zimmermant** [vpython/vpython-jupyter#17]
I was creating a fresh environment today, which installed JupyterLab 1.04.  When I tried installed the VPython extension I get the following:

```
An error occured.
ValueError: This extension does not yet support the current version of JupyterLab.
Conflicting Dependencies:
JupyterLab              Extension      Package
>=1.0.2 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/application
>=1.0.2 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/apputils
>=1.0.2 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/notebook
```
This isn't a big deal for me since I've got other environments installed that work but I thought I'd mention it in case this causes problems for others in the future.

**Comment by jcoady** [comment]
I will need to build a new version of vpython labextension on npm and bump the version number in order to fix this. The current version of vpython labextension works with jupyterlab version 0.35.3. However they have released jupyterlab version 1.0.2 on conda so I will need to change one of the vpython labextension config files to indicate that it is for jupyterlab versions between 1.0.2 and 2.0.0 . I think there is a new version of vpython that will be coming out soon so when that is released a new version of vpython labextension should also be released with all the latest vpython updates. 

John

**Comment by mwcraig** [comment]
Fixed by #26
